logbook-servlet/src/main/java/org/zalando/logbook/servlet/LocalResponse.java

```diff
@@ -54,11 +54,6 @@ public String getProtocolVersion() {
     }
 
     @Override
-    public String getContentType() {
-        return Optional.ofNullable(super.getContentType()).orElse("");
```

Comment:
What it did before was to change `null` to `""`. The not obvious issue is/was that `LocalResponse` implements two different interfaces, `org.zalando.logbook.BaseHttpMessage` and `javax.servlet.http.HttpServletResponse`, and both share a common method `String getContentType()`. By converting `null` to `""` we accidentially changed the behaviour of the later one because from what I can tell the container (or the framework?) checks whether the content type was set by checking for `null`. 

I have no clue how to test this other than running the example provided by @wyzssw.